HPC/WRF-v422-patch-correctness.diff

```diff
@@ -0,0 +1,119 @@
+diff --git a/arch/configure.defaults b/arch/configure.defaults
+index 692fbfe9..5a465e91 100644
+--- a/arch/configure.defaults
++++ b/arch/configure.defaults
+@@ -775,6 +775,8 @@ LD              =       $(FC)
+ RWORDSIZE       =       CONFIGURE_RWORDSIZE
+ PROMOTION       =       #-fdefault-real-8
+ ARCH_LOCAL      =       -DNONSTANDARD_SYSTEM_SUBR  -DWRF_USE_CLM
++# to validate WRF correctness on aarch64 against x86_64; compile with x86 this alternate ARCH_LOCAL option; able to achieve identical output model on ubuntu 18.04 with gcc 10.2 (on graviton and intel processors)
++ARCH_LOCAL      =       -DNONSTANDARD_SYSTEM_SUBR  -DWRF_USE_CLM -DAARCH64_X86_CORRECTNESS_FIX
```

Comment:
isn't this upstream by now?

Reply:
why patch in the ifdef vs just switching the code to the a non-ifdefed version

Reply:
Yes, do we need this patch? Isn't it default now in mainline?

Reply:
Will we recommend that anyone run the older version?  Is there anything truly special about 4.2.2 compared to other versions?

Reply:
4.2.2 is the version that the customer was using when they first compains about WRF output model correctness on Graviton.  Here I provided instructions to both versions to allow people to repeat this experiment in case they are interested.